flask/app/models.py

```diff
@@ -44,8 +44,11 @@ def get_id(self):
 class Group(db.Model):
     group_id: int = db.Column(db.Integer, primary_key=True)
     group_code: str = db.Column(
-        db.String(50), nullable=False, unique=True
-    )  # i.e. CHEO, SK, AB
+        db.String(50),
```

Comment:
The maximum length can be checked by the width of this field instead of the constraint since `LENGTH(group_code) < 64` will always be true

Reply:
resolved in 42b09e1